src/main/kotlin/no/nav/amt/arena/acl/processors/DeltakerProcessor.kt

```diff
@@ -75,6 +70,26 @@ open class DeltakerProcessor(
 		metrics.publishMetrics(message)
 	}
 
+	private fun externalDeltakerGuard(arenaDeltakerRaw: ArenaDeltaker) {
```

Comment:
Her må du vel gjøre det samme for hist-processoren?

Reply:
Men jeg forstår ikke helt hva vi gjør og hvorfor :/ Er det bare en sjekk for å se at vi ikke mapper om en id til noe annet enn den amt-iden vi allerede har..?

Reply:
Hadde vært fint å ha på hist deltakerprosessoren også men det er ikke mulig fordi disse inneholder ikke EKSTERN_ID. 

Dette startet med at jeg ikke fant igjen id mapping for deltakere i translation tabellene, hvor jeg etter hvert innså at det er jo fordi det er komet deltakere og det blir da kastet exception uten å lagre mappingen. 

Det at vi ikke har denne iden gjør det vanskelig å finne igjen dataene i arena_data som samsvarer med våre komet deltakere og det gjør det da vanskelig å feilsøke problemer med at vi har en versjon av en deltaker og arena har noe annet, som påvirker de som fortsatt leser fra arena.

Reply:
er du sikker på at hist-deltakerne ikke har med eksternid? mener jeg sjekket det og fikk det bekreftet :/

Reply:
Jeg har sett igjennom flere nå og finner ingen med ekstern id. For eksempel den på id=523068 har ekstern id satt i arena, mod_user er oss og DELTAKERTYPEKODE er EKSTERN

Reply:
Jeg ser at du har rett i at de kan inneholde ekstern id, så har lagt til samme for hist deltakere :)